.orchestra/ci/install-dependencies.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-set -e
+set -euo pipefail
 
-SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+SCRIPT_DIR=$(realpath "$(dirname "${BASH_SOURCE[0]}")")
```

Comment:
We use the old one everywhere. If you have a reason to change, you have to change it everywhere.

Reply:
Dropped